packages/core/src/shapes/Polyline.ts

```diff
@@ -15,7 +16,9 @@ export interface PolylineProps<T>
     Stroke<T>,
     Fill<T>,
     Scale<T>,
-    Poly<T> {}
+    Poly<T> {
+  strokeLinecap: StrV;
```

Comment:
Could you explain why this is only in `Polyline` and not in `Stroke`?

Reply:
Wasn't too sure about adding them to other `Stroke` shapes like `Path` and `Polygon` because they can be closed paths. From the [docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linecap) this seems to be fine. Do you want me to add it to `Stroke` instead?

Reply:
Either way is fine with me! If you don't add it to `Stroke` then could you write your reasoning (as you've written here) as a code comment on the `strokeLinecap` attribute?

Also, unrelated but now that I think about it, don't we also need to remove this from our passthrough JSON file? (I forget where that is...)

Reply:
Don't think I'm aware of this file?

Reply:
Found some docs on this property. Looks like only `Line`, `Polyline` and `Path` can be affected by this prop. So I'm not adding this to all stroke-able shapes. Added some comments in the PR desc and [08c284d](https://github.com/penrose/penrose/pull/1423/commits/08c284d90c877af29d5948e0f38f93db17720e5a)